README.md

```diff
@@ -2,22 +2,22 @@
 
 **About this branch**
 
-This branch of MusicBot is only compatible with the `rewrite` version of the discord.py library, which can be installed using `python3 -m pip install -U git+https://github.com/Rapptz/discord.py@rewrite#egg=discord.py[voice]`. It is also a dependency in `requirements.txt`, so should be automatically installed by the bot if you run `update.py`.
+This branch of MusicBot is only compatible with the `rewrite` version of the discord.py library, which can be installed using `python3 -m pip install -U git+https://github.com/Rapptz/discord.py@rewrite#egg=discord.py[voice]`. Since it is a dependency in `requirements.txt`, the library should be automatically installed with running `update.py`.
```

Comment:
if we're changing it from `by the bot if you run`, let's make sure it sounds grammatically pleasing. I think a more appropriate replacement here is, `Since it is a dependency in; `requirements.txt`, the library will be automatically installed upon running `update.py`.